build-system/tasks/runtime-test/helpers.js

```diff
@@ -182,22 +182,21 @@ function karmaBrowserStart_() {
  * @return {!Promise<number>}
  */
 async function createKarmaServer(config) {
-  let resolver, browsers_, results_;
+  let resolver, results_;
   const deferred = new Promise((resolverIn) => {
     resolver = resolverIn;
   });
 
   const karmaServer = new Server(config, async (exitCode) => {
-    await reportTestRunComplete(browsers_, results_);
+    await reportTestRunComplete(results_);
```

Comment:
This noop doesn't feel like a noop because it only functions if you consider the cross-file changes... I'd split this change into a separate PR

Reply:
I recently wrote this code and can confirm that `browsers` is not used in any way, so okay with this change here. (The sentiment of Daniel's comment is worth following in other non-obvious cases though.)